docs_user/modules/proc_preparing-block-storage-service-by-customizing-configuration.adoc

```diff
@@ -13,16 +13,16 @@ The high level explanation of the tailor-made approach is:
 endif::[]
 
 .Procedure
```

Comment:
Do users need to follow the steps listed here in this exact order?

Reply:
Not really, I think the other is not that important.